nss-day-cohort-52/rare-api-smoky-pythons
# Added create and update methods on the PostView class

### Issue ticket(s): #9 #15 

### New file(s):

### Modified File(s): `models/post.py`, `views/post.py`

### Modifications:
- Added create and update methods to the `PostView` class.
- Added the `blank=True` attribute to the ImageURLField so that an image url can be optional on the front end.

### Steps to test:
1. While in the terminal, use git fetch --all in the root directory of this repo
2. Checkout out to `al-posts-create-edit`
3. Start your server and the client
4. Navigate to the PostEntry component and fill out the form to create a new post
5. On submit, you should be redirected back to the PostList component
6. Try editing a post by clicking on the edit button and then submit that edit
